package.json

```diff
@@ -73,9 +72,8 @@
     "ember-disable-prototype-extensions": "^1.1.2",
     "ember-export-application-global": "^2.0.0",
     "ember-faker": "^1.5.0",
-    "ember-fetch": "^6.4.0",
     "ember-load-initializers": "^2.0.0",
-    "ember-math-helpers": "^2.10.0",
+    "ember-math-helpers": "~2.11.3",
```

Comment:
This was required since `ember-math-helpers` bumps major version number of `ember-cli-htmlbars` in minor releases. To continue supporting Ember 2.x via bower (as we do in Ember try), 2.11.x is the last version of `ember-math-helpers` this codebase can depend on.